scripts/files/files_helper.py

```diff
@@ -5,5 +5,11 @@ def get_file_name_from_path(path: str) -> str:
     return os.path.basename(path)
 
 
+def strip_extension(file: str) -> str:
```

Comment:
You should also take account of `.TIF` and `.TIFF`
To avoid a `if` I would suggest doing something like:
```python
>>> extension = ".tif"
>>> stripped = "my_file.TIFF".lower().split(extension, 1)[0]
>>> print(stripped)
my_file
```

Reply:
I believe that with the changes I've added to https://github.com/linz/topo-imagery/pull/92 this function can be deleted, I'll do that once that PR is merged and I can rebase